### PR TITLE
Add `type` to script tag

### PR DESCRIPTION
**Pull request by ulrikstrid**

### CODE REVIEW

**Review by Drup — CHANGES_REQUESTED** [review]
Oh no, I just realized that I had a review on suspend that I never pushed? Sorry about that.

I don't like the convention so much, We should pick more consistent names.

**Comment by Drup** [comment]
@ulrikstrid I'm planing a release soon, do I have time to make the changes to get that in ?

**Comment by ulrikstrid** [comment]
As discussed "offline", I'll try to get to it within a week.

File: lib/html_types.mli [hunk 1]
```diff
@@ -2249,7 +2249,7 @@ type script = [ | `Script ]
 
 type script_attrib =
   [ | common | subressource_integrity
-    | `Async | `Charset | `Src | `Defer | `Mime_type
+    | `Async | `Charset | `Src | `Defer | `Mime_type | `Type
```
[the remark this answers is not in the document]
Comment (ulrikstrid) [reply lib/html_types.mli:2252]: Do I understand correctly that I should remove my new variant and change `` `Mime_type `` to `` `Script_type? ``

**Comment by cemerick** [comment]
Looks like this has gone stale; if the patch is okay otherwise, I could try to rebase, etc.?

**Comment by ulrikstrid** [comment]
Feel free to take this @cemerick

**Pull request closed without merging**